File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.js
function solution(arr1, arr2) {
    var answer = Array.from({length: arr1.length}, (a,_) => Array(arr2[0].length));
    
    for(let i=0;i<arr1.length;i++){
        for(let a=0;a<arr2[0].length;a++){
            let sum = 0;
            for(let j=0;j<arr1[0].length;j++){
                sum += arr1[i][j] * arr2[j][a];
            }
            answer[i][a] = sum;
        }
    }
    
    return answer;
}

File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.java
class Solution {
    public int[][] solution(int[][] arr1, int[][] arr2) {
        int row = arr1.length;
        int col = arr2[0].length;
        int common = arr1[0].length;
        
        int[][] answer = new int[row][col];
        
        for(int a=0;a<row;a++){
            for(int b=0;b<col;b++){
                for(int c=0;c<common;c++){
                    answer[a][b] += arr1[a][c] * arr2[c][b];
                }
            }
        }
        return answer;
    }
}